tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -154,6 +154,10 @@ protected override bool Skip (Type type, string protocolName)
 				case "AVMutableComposition":
 				case "AVCompositionTrackSegment":
 				case "MKMapSnapshotOptions":
+				case "NSLayoutAnchor`1":
+				case "NSLayoutDimension":
+				case "NSLayoutXAxisAnchor":
+				case "NSLayoutYAxisAnchor":
```

Comment:
Indeed those types also exist in `AppKit`, thanks.

However this isn't what's causing the mac tests to fail.